variables.tf

```diff
@@ -1,11 +1,13 @@
 variable "stack_config_local_path" {
   type        = string
   description = "Path to local stack configs"
+  default     = "examples/complete/stacks"
```

Comment:
This is really confusing. `stack_config_local_path` should be required unless it is ignored in some situation, in which case it should be documented as such.